### ISSUE TITLE
[Epic] Address Dev Portal UI Accessibility Issues - Part 2

### ISSUE THREAD

**Issue by kcbabo** [solo-io/gloo-portal-issues#197]
This epic tracks resolution of longer term accessibility issues found based on an Accessibility analysis of Dev Portal UI. Each of these should be spun out into a specific issue when picked up.

- [ ] skip link, table positioning
- [x] #198
- [ ] text contrast, search results
- [ ] support user-provided alt text for images as an override for default alt text
